design-system/Text/Text.presets.ts

```diff
@@ -17,7 +17,9 @@ export type IPresets =
   | "bigBold"
   | "formHelper"
   | "formLabel"
-  | "emojiSymbol";
+  | "emojiSymbol"
+  | "bigEmojiSymbol"
+  | "clickableText";
```

Comment:
Are these used A LOT in the app or in the Figma? If not I wouldn't add them and instead pass a `style` props. And if the only diff in the text is that it's bold we even have a `bold` props I think or `weight="bold"`

Reply:
@thierryskoda I removed presets that were hardly used :)